src/main/scala/TooHot.scala

```diff
@@ -0,0 +1,9 @@
+object TooHot extends App{
+
+  def climate (temperature:Int, isSummer:Boolean):Unit = {
+    if (isSummer==true && temperature > 60 && temperature<=100) print("true")
```

Comment:
isSummer is a boolean by itself so you don't need to do == true against it, you could do 
if (isSummer && temperature > 60 && temperature<=100)